src/main/java/org/testeditor/fixture/host/net/Connection.java

```diff
@@ -181,7 +181,7 @@ public Result doCommand(final String command) {
       out.flush();
       logger.debug(
           "*****************************************************************************************");
-      logger.debug("---> Command sent: \"{}\"", command);
+      logger.debug("---> Command sent: '{}'", command);
```

Comment:
Aside from this we should talk about logging conventions in the team (syntax, level ...)